examples/checkin_collection.sample

```diff
@@ -28,8 +28,8 @@
           "supplement": true,
           "location_id": "c21a334d-b11a-49ff-88d9-97a700ed263e",
           "material_type_id": "93da93b3-5c2c-4077-b92d-12341d2070d2",
-          "product_type": "ISBN",
-          "product_type_id": "31dcaba7-bce1-4a12-bd06-9ce901422285",
+          "product_id": "2573-0797",
+          "product_id_type": "ISSN",
```

Comment:
UUID

Reply:
This is a type? It can be ISBN or ISSN as per Jira description.

Reply:
I just picked it from Jira description example but yes it should be UUID. I have also updated the schema.